File: barcosEnContienda/src/gestorAplicacion/Personas/CapitanCivil.java
package gestorAplicacion.Personas;

import gestorAplicacion.Embarcaciones.Barco;

public class CapitanCivil extends Capitanes {
	public CapitanCivil(String apodo, int liderazgo, int elocuencia) {
		super(apodo, liderazgo, elocuencia);
	}
	
	public void yells() {
		System.out.print("civil");
	}
}

File: practica-2-grupo-1-equipo-7-Barcos/src/gestorAplicacion/personas/capitanCivil.py
from gestorAplicacion.embarcaciones.barco import Barco
from gestorAplicacion.personas.capitanes import Capitanes

class CapitanCivil(Capitanes):
    
    def __init__(self,apodo,liderazgo,elocuencia):
        super().__init__(apodo,liderazgo,elocuencia)

    def yells(self):
        return "Civil \n"